db/migrate/20180926020026_create_admin_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateAdminUsers < ActiveRecord::Migration
+  def change
+    create_table :admin_users do |t|
+
```

Comment:
this is pretty much an empty table, what was the intention with this? perhaps not for this PR maybe